src/kikan/events.py

```diff
@@ -27,19 +27,25 @@ def handle_input(self):
 
     def tick(self):
         self.notify('pre_update')
-        for entity in self.engine.game_world.entities:
+        for entity in self.engine.game_world.entities.values():
             entity._update()
         self.notify('update')
 
         self.handle_input()
 
-    def notify(self, event: EventType, ctx: EventContext = []) -> None:
```

Comment:
Why did you remove the `EventContext` type hints?

Reply:
Sorry, it's my IDE. I'll fix it